modules/channel-web/src/views/lite/components/Composer.tsx

```diff
@@ -91,6 +93,11 @@ class Composer extends React.Component<ComposerProps, { isRecording: boolean }>
       return null
     }
 
+    let direction
```

Comment:
couldn't you simply
```ts
const direction =  isRTLText.test(this.props.message) ? 'rtl' : 'ltr'
```

Reply:
@EFF I tried that first, but with a ternary statement we add a class even when the user hasn't typed anything, and it overrides the default direction. This way, when the user hasn't typed anything, `direction` is undefined, so no class is added, and we inherit the correct default style. 

Check out how the cursor is on the right before we begin typing here.
![Composer](https://user-images.githubusercontent.com/159949/142876143-ef7ac970-5a0f-4ac4-b1cb-103c12e8f1aa.gif)